### PR TITLE
chore: learn page completed

### PR DESCRIPTION
**Pull request by hdJerry**
As a user of the application, I would like the "learn" pages to be redesigned with the new styles

NOTE: the branch needs to be merged into a new branch called "redesign" otherwise it will break the live site!

AC:
- [x] Ensure the page has dark background and color applied
- [x] Redesign the "crypto" banner
- [x] Redesign the courses section (make sure that the cards have the share button in this page)
- [x] Redesign the tutorial section (layout with 3 column)
- [x] Make sure the correct typography and colours are used

## output

Blockchain Developer Hub

### CODE REVIEW

File: src/sections/LearnCrypto/LearnCrypto.jsx [hunk 1]
```diff
@@ -1,27 +1,23 @@
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import Card from '@/components/Card';
-import TutorialCard from '@/components/TutorialCard';
+// import TutorialCard from '@/components/TutorialCard';
```
Comment (vyktoremario) [inline src/sections/LearnCrypto/LearnCrypto.jsx:4]: Will this be needed?
If not remove it

**Review by vyktoremario — COMMENTED** [review]
LGTM. But can I see the mobile view cos this changes a lot of styling.
And also the comment I left.

File: src/sections/LearnCrypto/LearnCrypto.jsx (hunk 1 above)
Comment (hdJerry) [reply src/sections/LearnCrypto/LearnCrypto.jsx:4]: Thanks chief

**Review by vyktoremario — APPROVED** [review]
ship it 🚀

**Pull request merged**